### DISCUSSION TITLE
Ghost role raffle adjustments

### DISCUSSION THREAD

**Discussion by pissdemon** [DeltaV-Station/Delta-v#1304] [General]
Some ghost role raffle timers probably need adjustments - maybe the raffle duration is too long, maybe it is too short, maybe something should be raffled or not raffled. Put your suggestions and comments below on what should be changed, and to what.

**Comment by Guess-My-Name** [comment]
Xenos should be changed from default to short - 30 seconds for xenos is way too long, they are as squishy as slimes and will be dead before timer is up if they spawn in a public area
And imo the default one could get it's joinExtendsDurationBy changed from 10 to ~5-7, because if there are even 5 ghosts interested in a rev, currently the timer for spawn will increase to over a minute, unnecessarily long.
